DQMOffline/Lumi/plugins/ZCounting.cc

```diff
@@ -388,8 +440,222 @@ void ZCounting::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
 
   }//End of tag loop
 
+}
+void ZCounting::analyzeElectrons(const edm::Event& iEvent, const edm::EventSetup& iSetup) {
+  edm::LogInfo("ZCounting") <<  "ZCounting::analyzeElectrons" << std::endl;
+
+  //-------------------------------
+  //--- Vertex
+  //-------------------------------
+  edm::Handle<reco::VertexCollection> hVertexProduct;
+  iEvent.getByToken(fPVName_token,hVertexProduct);
+  if(!hVertexProduct.isValid()) return;
+
+  const reco::VertexCollection *pvCol = hVertexProduct.product();
+  int nvtx = 0;
+
+  for(reco::VertexCollection::const_iterator itVtx = pvCol->begin(); itVtx!=pvCol->end(); ++itVtx) {
```

Comment:
hi @AndreasAlbert - please make this a range-based loop. (eg, for ( const auto & vtx : *pvCol)
(seems I missed this before)